webboot/webboot.go

```diff
@@ -26,32 +33,102 @@ var (
 	ipv4      = flag.Bool("ipv4", true, "use IPV4")
 	ipv6      = flag.Bool("ipv6", true, "use IPV6")
 	osSystems = map[string]webboot.Distro{
-		"TinyCore": webboot.Distro{"/boot/vmlinuz64 ", "/boot/initrd", "--reuse-cmdline", "https:louis.com"},
-		"Ubuntu":   webboot.Distro{"/boot/vmlinuz ", "/boot/init", "--reuse-cmdline", "https:louis.com"},
+		"tinycore": webboot.Distro{"/boot/vmlinuz64 ", "/boot/initrd", "--reuse-cmdline", "http://tinycorelinux.net/10.x/x86_64/release/CorePure64-10.1.iso"},
+		"ubuntu":   webboot.Distro{"/boot/vmlinuz ", "/boot/init", "--reuse-cmdline", "https://releases.ubuntu.com/bionic/ubuntu-18.04.2-desktop-amd64.iso"},
 	}
 )
 
+//wgetArgs takes in the arg user passes e.g. name of ISO or URL to an ISO
+//returns an io.ReadCloser, URL link, filename, and err
+//io.ReadCloser and err is returned from wget
+// if the arg is a name of an URL found in the osSystems, the Download Link specificed to the name of ISO is used for wget & urlToName to generate a name
```

Comment:
That is true. I see what you mean by that, and I will try to explain the WHY in this comment

Reply:
I will explain how I am trying to return the valid args back to main which are the proper links and the proper filename for the content to be stored as, so it can be used correctly throughout the rest of the code.